poseObject: [0.555,0.437,0.071,0.0,0.0,0.0]

radiusObject: 0.0

poseReference: [0.10,0.10,0.15,3.1416,0.0,0.0]

radiusTrajectory: 0.0

//TO BE MOVED